src/boot/chid.h

```diff
@@ -36,6 +39,13 @@ typedef struct Device {
                         uint32_t name_offset;          /* nul-terminated string or 0 if not present */
                         uint32_t compatible_offset;    /* nul-terminated string or 0 if not present */
                 } devicetree;
+                struct {
+                        /* Offsets are relative to the beginning of the .hwids PE section.
+                         * They are nul-terminated strings when present or 0 if not present */
+                        uint32_t name_offset;       /* name or identifier for the firmware blob */
+                        uint32_t compatible_offset; /* compatibility identifier to match a specific fw blob */
```

Comment:
so, my understanding is that the "compatible" field here is supposed to be some string that we look for in the uefi firmware blob metadata, right? maybe call that fwid or so? 

i think thta would make things clearer, since this has nothing to do with the devicetree "compatible" string...

Reply:
> so, my understanding is that the "compatible" field here is supposed to be some string that we look for in the uefi firmware blob metadata, right?

Right. We had that hashed out in the other PR I sent.